include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -31,17 +31,20 @@ typedef struct _oe_sgx_enclave_image_info_t
 // oe_sgx_enclave_properties_t SGX enclave properties derived type
 #define OE_SGX_FLAGS_DEBUG 0x0000000000000002ULL
 #define OE_SGX_FLAGS_MODE64BIT 0x0000000000000004ULL
+#define OE_SGX_FLAGS_KSS 0x0000000000000080ULL
 #define OE_SGX_SIGSTRUCT_SIZE 1808
 
 typedef struct oe_sgx_enclave_config_t
 {
     uint16_t product_id;
     uint16_t security_version;
+    uint8_t family_id[16];
```

Comment:
The `padding` at line 45 is a padding for `product_id` and `security_version` to ensure variables do not go across 64-byte boundary. I'd put `padding` immediately after `product_id` and `security_version`.

Reply:
@CodeMonkeyLeet Yeah, I meant 64-bit aligned. Thanks for the correction.

Reply:
Good catch. Updated. Thanks